prosody/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apt-dpkg-wrap apt-get update && \
     cd /tmp/luarocks && ./configure && make && make install && cd - && \
     luarocks install cyrussasl 1.1.0-1 && \
     luarocks install net-url 0.9-1 && \
-    luarocks install luajwtjitsi 2.0-0
+    luarocks install luajwtjitsi 3.0-0
```

Comment:
Have you tested that? There is a jitsi-meet PR that is not merged yet ...

Reply:
No, but we cannot build the previous release with lua 5.4 due to a dependency problem, so the only way is forward!